### PR TITLE
 t-tabularToRelational - helpers 2.0.0

### PR DESCRIPTION
**Pull request by skodape**
Migration of DPU to helpers 2.0.0 DPU functionality has not been tested as it's not known to me.

Note: You may consider localization of about.html.

### CODE REVIEW

**Comment by skrchnavy** [comment]
@eea03 please check if this is valid

**Comment by eea03** [comment]
This was developed and also tested by @eea02, so I am reasigning to him

**Comment by eea02** [comment]
I have successfully tested it, we can merge it.

File: t-tabularToRelational/README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 |Version |Release notes |
 |--------|--------------|
+|1.0.1   |Update to helpers 2.0.0 |
```
Comment (skrchnavy) [inline t-tabularToRelational/README.md]: version shall be 1.1.0 (or 2.0.0)

File: t-tabularToRelational/pom.xml [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	<artifactId>uv-t-tabularToRelational</artifactId>
 	<name>T-TabularToRelational</name>
 	<description>Parse tabular file to relational data unit.</description>
-	<version>1.0.0</version>
+	<version>1.0.1-SNAPSHOT</version>
```
Comment (skrchnavy) [inline t-tabularToRelational/pom.xml]: sync with Readme.md, version 1.1.0 or 2.0.0

File: t-tabularToRelational/pom.xml [hunk 3]
```diff
@@ -50,7 +52,19 @@
 			<version>${module-test.version}</version>
 			<scope>test</scope>
 		</dependency>
-
+        <!-- UnifiedViews helpers. -->
```
Comment (skrchnavy) [inline t-tabularToRelational/pom.xml]: Unifications of tabs and spaces necessary

**Comment by tomas-knap** [comment]
Please address the minor issues Sveto mentioned, then you can merge.

**Pull request merged**